src/LanguageServer/Protocol.TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -108,7 +100,7 @@ private protected class OrderLocations : Comparer<LSP.Location?>
             public override int Compare(LSP.Location? x, LSP.Location? y) => CompareLocations(x, y);
         }
 
-        protected virtual TestComposition Composition => EditorFeaturesLspComposition;
+        protected virtual TestComposition Composition => FeaturesLspComposition;
```

Comment:
we generally don't need editorfeatures in the composition anymore.  Previously it was required to test LSP APIs that returned internal VS types - we would have a default service in the protocol layer, and override it in editor features (to return things like classified text runs).  

However now we have all our own types and the conversions are in the protocol layer too, so this is generally not necessary.